File: template/CodeSmith/CodeGenerator/01-Templates/02-Result/Presentation/Typings/app/dbo/teamRepositoryRole/model/teamRepositoryRole-query.ts
import { QueryParameter } from '../../../../util';

/**
 * 查询参数
 */
export class TeamRepositoryRoleQuery extends QueryParameter {
    /**
     * 
     */
    id;
    /**
     * 
     */
    teamID;
    /**
     * 
     */
    repositoryID;
    /**
     * 
     */
    allowRead;
    /**
     * 
     */
    allowWrite;
}

File: template/CodeSmith/CodeGenerator/01-Templates/02-Result/Services/Queries/dbo/TeamRepositoryRoleQuery.cs
using System;
using System.ComponentModel.DataAnnotations;
using Ding;
using Ding.Datas.Queries;

namespace GitDC.Service.Queries.dbo {
    /// <summary>
    /// 查询参数
    /// </summary>
    public class TeamRepositoryRoleQuery : QueryParameter {
        /// <summary>
        /// 
        /// </summary>
        [Display( Name = "" )]
        public Guid? Id { get; set; }
        /// <summary>
        /// 
        /// </summary>
        [Display( Name = "" )]
        public long? TeamID { get; set; }
        /// <summary>
        /// 
        /// </summary>
        [Display( Name = "" )]
        public long? RepositoryID { get; set; }
        /// <summary>
        /// 
        /// </summary>
        [Display( Name = "" )]
        public bool? AllowRead { get; set; }
        /// <summary>
        /// 
        /// </summary>
        [Display( Name = "" )]
        public bool? AllowWrite { get; set; }
    }
}